code/leslie/Python/lab02.py

```diff
@@ -1,22 +1,27 @@
-
-ones_digits = ["","one","two","three","four","five","six","seven","eight","nine", "ten", "eleven", "twelve", "thirteen", "fourteen", "fifteen", "sixteen", "seventeen", "eighteen", "nineteen"]
-tens_digits = ["", "ten", "twenty", "thirty", "forty", "fifty", "sixty", "seventy", "eighty", "ninety", ""]
-hundreds = ["", "one hundred","two hundred","three hundred", "four hundred","five hundred","six hundred", "seven hundred", "eight hundred", "nine hundred"]
-
+ones_digits = {0: "", 1: "one", 2: "two", 3: "three", 4: "four", 5: "five", 6: "six", 7: "seven", 8: "eight", 9: "nine", 10: "ten", 11: "eleven", 12: "twelve", 13: "thirteen", 14: "fourteen", 15: "fifteen", 16: "sixteen", 17: "seventeen", 18: "eighteen", 19: "nineteen"}
```

Comment:
* Suggestion:
    * Dictionaries can be spread over many lines to make them more readable. Consider that in the future:
        * Example:
```
ones_digits = {
    0: "",
    1: "one",
    2: "two",
    #...
}
```